python/lsst/ts/ofc/ofc.py

```diff
@@ -346,28 +281,27 @@ def reset(self):
     def set_pssn_gain(self):
         """Set the gain value based on the PSSN, which comes from the FWHM by
         DM team.
-
         Raises
```

Comment:
Why do you remove the empty line before the **Raises** section? There should be an empty line before it to follow the numpy doc string format.